File: java/spring-racuni/src/main/java/si/um/feri/rest/racuni/Racun.java
package si.um.feri.rest.racuni;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Racun {
    @Id
    private String id;
    private String datum;
    private String kraj;
    private double znesek;
    private String ppId;
    private String davcnaSt;
}

File: cs/MinimalApiRacuni/Models/Racun.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MinimalApiRacuni.Models
{
    public class Racun
    {
        public string? Id { get; set; }
        public string? Datum { get; set; }
        public string? Kraj { get; set; }
        public double Znesek { get; set; }
        public string? PpId { get; set; }
        public string? DavcnaSt { get; set; }

    }
}
